t/t5601-clone.sh

```diff
@@ -68,6 +68,14 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
+
+	mkdir UPPERCASE &&
+	lower=$(cygpath -w "$(pwd)/UPPERCASE" | tr A-Z a-z) &&
+	git clone src "$lower" &&
+	test -z $(git -c UPPERCASE config core.worktree)
```

Comment:
Oh, but that won't work without double quotes around the `$(...)`, will it?

Reply:
Works for me, but again I'm such a shell script newbie. I added double quotes, it also seems to work fine. I also reworked this test to avoid the possibility of other errors in `git config` to be incorrectly recognized as missing key.